lib/pipedrive/helpers/rate_limit.ex

```diff
@@ -35,7 +35,7 @@ defmodule Pipedrive.Helpers.RateLimit do
   end
 
   defp execute_request(api_call, api_call_args) do
-    case apply(api_call, api_call_args) do
+    case apply(api_call, List.wrap(api_call_args)) do
```

Comment:
All of the tests still pass if I remove the call to `List.wrap`. What is it for?

Reply:
It's just a cosmetic additions, it allows passing a single scalar as an argument (without wrapping it in a list), so with these both of these would work:

```elixir
sleep_and_retry(&Person.get/1, "person_id_123")
sleep_and_retry(&Person.get/1, ["person_id_123"])
```